examples/augmented_inference.jl

```diff
@@ -0,0 +1,82 @@
+using AbstractGPs
+using TemporalGPs
+using Distributions
+using StatsFuns
+using LinearAlgebra
+using ProgressMeter
+
+# Utilising TemporalGPs.jl to work with AbstractGPs requires minimal modification to the
```

Comment:
Could you replace this generic blurb with something specific to this script? Probably good to point to your paper, and point out algorithm 1 or something.